source/index.html.md

```diff
@@ -21,7 +21,7 @@ code_clipboard: true
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+Welcome to the Edgar API! You can use our API to access Edgar API endpoints, which can get information on various E D G A R in our database.
```

Comment:
This is an example of a code review. Everything you wrote looks good here. Change one more place in the code as well.

Reply:
Added some crazy!

Reply:
http://54.146.92.27:4567/?shell#introduction
It's Up!